package.json

```diff
@@ -333,6 +333,29 @@
                         "type": "string",
                         "default": "",
                         "description": "Defines which files in the workspace to ignore for Zip deploy. This applies to Zip deploy only, has no effect on other deployment methods."
+                    },
+                    "azureFunctions.projectRuntime": {
+                        "type": "string",
+                        "enum": [
+                            "~1",
```

Comment:
I thought there was a ~2 available?

Reply:
I thought so, too, but it has weird behavior.

![screen shot 2017-12-14 at 11 14 11 am](https://user-images.githubusercontent.com/11282622/34010013-f78762b2-e0bf-11e7-8c9b-218e2da53b7a.png)

'~2' is _supposed_ to mean v2, but that message says it's older than v1. It's possible '~2' won't be supported as a version until 'beta' is GA'ed